### PR TITLE
Fixed requirements

### PR DESCRIPTION
**Pull request by invinciblycool**
#### Short description of what this resolves:

Fixes the failing travis build.

#### Changes proposed in this pull request:

- Changed pyscopg2 version to 2.7.3.1

### CODE REVIEW

**Comment by RaiVaibhav** [comment]
@invinciblycool  whole time i was thinking why it its showing that requirement error

**Comment by ananyo2012** [comment]
@invinciblycool Thanks for the fix. I was too wondering what was causing the Travis build failure.

**Comment by RaiVaibhav** [comment]
@invinciblycool i think just removing the 2.7 making its runs fine i.e only `>=2.6.1`

**Comment by invinciblycool** [comment]
@ananyo2012 welcome.
@RaiVaibhav, try running it on your fork with Travis, if the build is successful I will amend my commit.

**Comment by RaiVaibhav** [comment]
@invinciblycool  its working fine check the log of [PR](https://github.com/fossasia/open-event-orga-server/pull/4507)

File: requirements/common.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ Flask-JWT>=0.3.2,<0.4
 requests-oauthlib>=0.7.0,<1
 icalendar>=3.11,<4
 requests[security]>=2.12.4,<3
-psycopg2>=2.6.2,<2.7
+psycopg2==2.7.3.1
```
Comment (niranjan94) [inline requirements/common.txt]: Don't pin it to this version. Make it `>=2.7,<2.8`

**Comment by invinciblycool** [comment]
@niranjan94 and @RaiVaibhav made the changes as requested.

**Pull request merged**